pkg/controller/lifecycle/actuator_test.go

```diff
@@ -240,8 +240,7 @@ var _ = Describe("Actuator", func() {
 					expectedSeedServiceMonitor(namespace),
 				))
 			},
-			Entry("Kubernetes version < 1.26", semver.MustParse("1.25.0"), false, false, false, false),
-			Entry("Kubernetes version >= 1.26", semver.MustParse("1.26.0"), true, false, false, false),
+			Entry("Kubernetes version >= 1.27", semver.MustParse("1.27.0"), true, false, false, false),
```

Comment:
The `k8sVersion` parameter is becoming obsolete, it was used just to generated different pod disruption budgets depending on whether the k8s version `<= v1.26` or not.

For completeness, you can also drop the `k8sVersion` parameter from the `getSeedResources` function here:
- https://github.com/gardener/gardener-extension-shoot-lakom-service/blob/9ee7121684091d71148300c087a2341c929cb111/pkg/controller/lifecycle/actuator.go#L290

Then the PDB can be changed to always set `unhealthyPodEvictionPolicy=AlwaysAllow` here
- https://github.com/gardener/gardener-extension-shoot-lakom-service/blob/9ee7121684091d71148300c087a2341c929cb111/pkg/controller/lifecycle/actuator.go#L496-L508


`expectedSeedPDB` can be also simplified as now it is always the same
- https://github.com/gardener/gardener-extension-shoot-lakom-service/blob/9ee7121684091d71148300c087a2341c929cb111/pkg/controller/lifecycle/actuator_test.go#L560-L587